orange/src/main/java/com/hartwig/hmftools/orange/algo/sigs/SigsInterpreter.java

```diff
@@ -0,0 +1,33 @@
+package com.hartwig.hmftools.orange.algo.sigs;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import com.hartwig.hmftools.datamodel.sigs.ImmutableSignatureAllocation;
+import com.hartwig.hmftools.datamodel.sigs.SignatureAllocation;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SigsInterpreter
+{
+    @Nullable
+    public static List<SignatureAllocation> convertAndAnnotateWithEtiology(
```

Comment:
I would rename function to `interpret' to be more compatible with e.g. PurpleInterpreter